src/claim/definitions.js

```diff
@@ -0,0 +1,455 @@
+    version: '1',
+    type: {
+      properties: [
+        {
+          name: 'country',
+          type: 'claim-cvc:PhoneNumber.country-v1',
+        },
+        {
+          name: 'countryCode',
+          type: 'claim-cvc:PhoneNumber.countryCode-v1',
+        },
+        {
+          name: 'number',
+          type: 'claim-cvc:PhoneNumber.number-v1',
+        },
+        {
+          name: 'extension',
+          type: 'claim-cvc:PhoneNumber.extension-v1',
+        },
+        {
+          name: 'lineType',
+          type: 'claim-cvc:PhoneNumber.lineType-v1',
+        },
+      ],
+      required: ['country', 'countryCode', 'number', 'lineType'],
+    },
+    credentialItem: true,
+  },
+  {
+    identifier: 'claim-cvc:PhoneNumber.country-v1',
+    type: 'cvc:Type:country',
+    version: '1',
+    credentialItem: true,
+  },
+  {
+    identifier: 'claim-cvc:Type.Name-v1',
+    version: '1',
+    type: {
+      properties: [
+        {
+          name: 'givenNames',
+          type: 'claim-cvc:Name.givenNames-v1',
+        },
+        {
+          name: 'familyNames',
+          type: 'claim-cvc:Name.familyNames-v1',
+        },
+        {
+          name: 'otherNames',
+          type: 'claim-cvc:Name.otherNames-v1',
+        },
+      ],
+      required: ['givenNames'],
+    },
+    credentialItem: true,
+  },
+  {
+    identifier: 'claim-cvc:Type.address-v1',
+    version: '1',
+    type: {
+      properties: [
+        {
+          name: 'country',
+          type: 'claim-cvc:Address.country-v1',
+        },
+        {
+          name: 'county',
+          type: 'claim-cvc:Address.county-v1',
+        },
+        {
+          name: 'state',
+          type: 'claim-cvc:Address.state-v1',
+        },
+        {
+          name: 'street',
+          type: 'claim-cvc:Address.street-v1',
+        },
+        {
+          name: 'unit',
+          type: 'claim-cvc:Address.unit-v1',
+        },
+        {
+          name: 'city',
+          type: 'claim-cvc:Address.city-v1',
+        },
+        {
+          name: 'postalCode',
+          type: 'claim-cvc:Address.postalCode-v1',
+        },
+      ],
+      required: ['street', 'unit', 'city', 'state', 'country'],
+    },
+    credentialItem: true,
+  },
+  {
+    identifier: 'claim-cvc:Address.street-v1',
+    version: '1',
+    type: 'String',
+    credentialItem: true,
+  },
+  {
+    identifier: 'claim-cvc:Address.unit-v1',
+    version: '1',
+    type: 'String',
+    credentialItem: true,
+  },
+  {
+    identifier: 'claim-cvc:Document.properties-v1',
+    version: '1',
+    attestable: true,
+    type: {
+      properties: [
+        {
+          name: 'dateOfIssue',
+          type: 'claim-cvc:Document.dateOfIssue-v1',
+        },
+        {
+          name: 'dateOfExpiry',
+          type: 'claim-cvc:Document.dateOfExpiry-v1',
+        },
+      ],
+      required: ['dateOfIssue'],
+    },
+    credentialItem: true,
+  },
+  {
+    identifier: 'claim-cvc:Type.S3DocumentImageRef-v1',
```

Comment:
Not a Claim

Reply:
Needed this in order to override `S3DocumentImageRef` because it has a property that is a claim: claim-cvc:Document.type-v1 (it's marked as `credential-item: true`).
Is `cvc:Document:type` a claim?

Reply:
Solved